pkg/db/db.go

```diff
@@ -108,20 +153,12 @@ func (o Organization) CurrentPublicKey() (jwk.Key, error) {
 		return nil, errors.New("organization has no active certificates")
 	} else {
 		// Organization has no certificates, fallback to plain JWKs
-		if len(o.Keys) > 0 {
-			set, err := o.KeysAsSet()
-			if err != nil {
-				return nil, err
-			}
-			key := set.Keys[0]
-			return key, nil
-		}
-		// Organization has no JWKs, fallback to PublicKey
-		key, err := crypto.PemToPublicKey([]byte(*o.PublicKey))
+		set, err := o.KeysAsSet()
 		if err != nil {
 			return nil, err
 		}
-		return jwk.New(key)
+		key := set.Keys[0]
```

Comment:
Are the keys ordered? Should you check time validity here as well?

Reply:
Good question since a 'set' is generally orderless, I'll check